File: C/1013.c
#include <stdio.h>

int main ()
{
	int A, B, C, maior;
	scanf ("%d %d %d", &A, &B, &C);
	maior = (A + B + abs(A - B)) / 2;
	if (maior > C)
		printf ("%d eh o maior\n", maior);
	else
		printf ("%d eh o maior\n", C);
	return 0;
}

File: JavaScript/1013.js
var lines = input.split('\n');

const firstLine = lines.shift().split(' ');

function comparar(a,b){
    return (a+b+Math.abs(a-b))/2
}

function maior(a,b,c){
    var comparado = comparar(a,b) 
    return comparado > c ? comparado : c
}

var firstInput = parseInt(firstLine.shift());
var secondInput = parseInt(firstLine.shift());
var thirdInput = parseInt(firstLine.shift());

console.log(maior(firstInput,secondInput,thirdInput) + " eh o maior");
